packages/components/src/form-token-field/types.ts

```diff
@@ -154,6 +158,10 @@ export interface FormTokenFieldProps
 	 * @default false
 	 */
 	__next36pxDefaultSize?: boolean;
+	/**
+	 * Custom renderer for suggestions.
+	 */
+	__experimentalRenderItem?: ( args: { item: string } ) => ReactNode;
 }
 
 export interface SuggestionsListProps< T = string | { value: string } > {
```

Comment:
I believe this now needs to be an inexact object:

```suggestion
export interface SuggestionsListProps<
	T = string | ( Record< string, unknown > & { value: string } )
> {
```

Reply:
I agree. Thanks